iotcore/src/main/java/com/google/android/things/iotcore/IotCoreClient.java

```diff
@@ -232,16 +244,27 @@ private IotCoreClient(
         mClientConnectionState = clientConnectionState;
 
         mMqttClient.setCallback(
-                createMqttCallback(onConfigurationExecutor, onConfigurationListener));
+                createMqttCallback(onConfigurationExecutor, onConfigurationListener, onCommandExecutor, onCommandListener));
+
+        // Always subscribe to the device configuration topic
```

Comment:
like commands, if configuration listener is null, it would be not necessary to subscribe MQTT configuration topic.

Reply:
true, will change that